readme.txt

```diff
@@ -1,8 +1,8 @@
 === New Relic Reporting for WordPress ===
 Contributors:      10up, rittesh.patel, tott, oscarssanchez, jeffpaul
 Tags:              New Relic, New Relic Reporting, New Relic APM Reporting, APM Reporting
-Requires at least: 4.0
-Tested up to:      6.3
+Requires at least: 6.3
+Tested up to:      6.5.2
```

Comment:
For future reference, you can leave the major version here (e.g. 6.5) and WP.org will update to show the current minor as the tested-up-to version.